packages/fxa-auth-server/test/local/senders/emails.ts

```diff
@@ -873,16 +873,16 @@ const TESTS: [string, any, Record<string, any>?][] = [
   ])],
 
   ['verifyLoginCodeEmail', new Map<string, Test | any>([
-    ['subject', { test: 'equal', expected: 'Sign-in code for Mock Relier' }],
+    ['subject', { test: 'equal', expected: 'Approve sign-in to Mock Relier' }],
     ['headers', new Map([
       ['X-SES-MESSAGE-TAGS', { test: 'equal', expected: sesMessageTagsHeaderValue('verifyLoginCode') }],
       ['X-Signin-Verify-Code', { test: 'equal', expected: MESSAGE.code }],
       ['X-Template-Name', { test: 'equal', expected: 'verifyLoginCode' }],
       ['X-Template-Version', { test: 'equal', expected: TEMPLATE_VERSIONS.verifyLoginCode }],
     ])],
     ['html', [
-      { test: 'include', expected: 'Sign-in code for Mock Relier' },
-      { test: 'include', expected: 'Is this you signing in?' },
+      { test: 'include', expected: 'Approve sign-in to Mock Relier' },
+      { test: 'include', expected: 'Did you sign in to Mock Relier?' },
```

Comment:
needs to be hyphenated here and below too, but the tests will fail once/if we modify reference string anyways.